Modules/_functoolsmodule.c

```diff
@@ -24,9 +24,36 @@ typedef struct {
     vectorcallfunc vectorcall;
 } partialobject;
 
-static PyTypeObject partial_type;
+typedef struct _functools_state {
+    /* this object is used delimit args and keywords in the cache keys */
+    PyObject *kwd_mark;
+    PyTypeObject *partial_type;
+    PyTypeObject *lru_cache_type;
```

Comment:
`lru_cache_type` is unused and can be removed from the C-level state.

Reply:
You mean this line can be removed too?
https://github.com/python/cpython/blob/master/Lib/functools.py#L642

Reply:
No. But that line that uses the module's `__dict__`, not the C-level state.

Reply:
Oh. Thanks, Got you means of your comment : )